extension/src/time_weighted_average.rs

```diff
@@ -48,7 +48,7 @@ impl<'input> TimeWeightSummary<'input> {
         prev: Option<TimeWeightSummary>,
         next: Option<TimeWeightSummary>,
     ) -> TimeWeightSummary<'static> {
-        assert!(interval_start <= self.first.ts && interval_start + interval_len >= self.last.ts);
+        assert!(interval_start <= self.first.ts && interval_start + interval_len > self.last.ts);
```

Comment:
This is just fixing the assert to reflect that the aggregate is exclusive on the upper bound.  If this assert fails, then the caller is trying to call an interpolated function using an interval that's smaller than their bucketing interval.  It might still be useful to add a test that we handle invalid input correctly, but `pg_test` still has trouble with failure tests.